README.md

```diff
@@ -50,7 +55,9 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+cat = town[:residents].select {|character| character == "Belle"}
+town[:castle][:guests].push(cat)
```

Comment:
pry won't let you use `cat` as a local variable (fun fact! this should work in most other contexts) but cool approach.

`town[:residents].select {|character| character == "Belle"}` returns an array with one element also

you'd have to delete Belle from town[:residents also] but 👍 overall